src/sentry/api/endpoints/organization_events.py

```diff
@@ -143,7 +144,25 @@ def data_fn(offset, limit):
                     "error": six.text_type(error),
                 },
             )
-            raise ParseError(detail="Invalid query.")
+            message = "Internal error. Please try again."
+            if type(error) in [
+                snuba.RateLimitExceeded,
```

Comment:
Could we set the message where these errors are defined in `sentry.utils.snuba` instead, so that we'll have the same messages elsewhere?

Reply:
I didn't want to do that because the original error could have a bunch of useful internal snuba info (e.g. the actual sql query being run) and I didn't want to overwrite it until we were sending it to the client.